airflow-core/src/airflow/api_fastapi/core_api/openapi/_private_ui.yaml

```
@@ -1872,6 +1872,7 @@ components:
     MenuItem:
       type: string
       enum:
+      - Action Required
```

Maybe rename this (constantly over all code base)?
```suggestion
      - Required Actions
```

---

Reorder the adjective to come first for better consistency for me, too The update has been applied.